app/ui-react/pom.xml

```diff
@@ -452,6 +388,139 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>non-ui-disabled</id>
```

Comment:
I don't want to be picky here, but can we either make the id more clear in its intentions or add an xml comment. I cannot really wrap my mind around what `non-ui-disabled` actually means

Reply:
How about `non-ui-related-so-disabled`?

Reply:
definitely better!